website_sale_product_brand/README.rst

```diff
@@ -0,0 +1,95 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================
+Website Sale Product Brand
+==========================
+
+Description
+===========
+
+This module was written to extend the functionality of product filtering on website.
+It will allow you to filter product based on its brand.
+
+While shopping online, we have seen various eShops having a feature to shop by brands
+which ODOO does not yet provide officially.Website Sale Product Brand fills the gap at certain
+extent and by providing basic search by brands, thus reducing end-user’s efforts in 
+searching the products he/she wants to purchase.
+
+Installation
+============
+
+To install this module, you need to install following module:
+->https://github.com/OCA/product-attribute/tree/11.0/product_brand
```

Comment:
Dependencies are already documented in the manifest file

Reply:
That's not true. You have to indicate dependencies outside of the current repo, which is the case.